functions/find_gif.ts

```diff
@@ -1,5 +1,5 @@
 import { DefineFunction, Schema, SlackFunction } from "deno-slack-sdk/mod.ts";
-import gifs from "../assets/gifs.json" with { type: "json" };
```

Comment:
The `with` import assertion fails when using deno 2 in combination w/ esbuild bundling (bundling happens when you run `slack deploy` using the CLI)